vismne/viz.py

```diff
@@ -203,3 +379,37 @@ def save_movie(self):
     def animate(self):
         """Doc."""
         raise NotImplementedError
+
+
+class OverlayData(object):
+    """Encapsulation of statistical neuroimaging overlay viz data."""
```

Comment:
needs to be public?

Reply:
Removed and turned into a private method `Brain._get_overlay_limits`